src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php

```diff
@@ -26,12 +28,14 @@ class="tribe-events-calendar-month__calendar-event-datetime-featured tribe-commo
 		>
 		</em>
 	<?php endif; ?>
-	<time datetime="<?php echo esc_attr( $event->dates->start->format( 'H:i' ) ) ?>">
-		<?php echo esc_html( $event->dates->start->format( $time_format ) ) ?>
-	</time>
-	<span class="tribe-events-calendar-month__calendar-event-datetime-separator"> - </span>
-	<time datetime="<?php echo esc_attr($event->dates->end->format( 'H:i' )) ?>">
-		<?php echo esc_html( $event->dates->end->format( $time_format ) ) ?>
+	<time datetime="<?php echo esc_attr( $event->dates->start->format( 'H:i' ) ); ?>">
+		<?php echo esc_html( $event->dates->start->format( $time_format ) ); ?>
 	</time>
+	<?php if ( $display_end_date ) : ?>
+		<span class="tribe-events-calendar-month__calendar-event-datetime-separator"> - </span>
+		<time datetime="<?php echo esc_attr($event->dates->end->format( 'H:i' ) ); ?>">
+			<?php echo esc_html( $event->dates->end->format( $time_format ) ); ?>
+		</time>
+	<?php endif; ?>
```

Comment:
`start_display`?

Reply:
Whenever we display an event date, we should use `$event->dates->(start|end)_display`.